app/src/main/java/framgia/com/ichat/data/source/PrivateRoomDataSource.java

```diff
@@ -1,15 +1,18 @@
-package framgia.com.ichat.data.source;
-
-import com.google.android.gms.tasks.OnCompleteListener;
-import com.google.android.gms.tasks.OnFailureListener;
-import com.google.firebase.database.ValueEventListener;
-
-public interface PrivateRoomDataSource {
-    interface Remote {
-        void getPrivateRooms(ValueEventListener valueEventListener);
-
-        void createPrivateRoom(OnCompleteListener onCompleteListener, OnFailureListener onFailureListener);
-
-        void deletePrivateRoom(String id, OnCompleteListener onCompleteListener, OnFailureListener onFailureListener);
-    }
-}
+package framgia.com.ichat.data.source;
+
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.firebase.database.ValueEventListener;
+
+public interface PrivateRoomDataSource {
+    interface Remote {
+        void getPrivateRooms(ValueEventListener valueEventListener);

```

Comment:
```suggestion
        void getRooms(ValueEventListener valueEventListener);
```
nên để là getRooms
vì nếu gọi PrivateRoomDataSource.Remote.getPrivateRooms thì duplicate chữ "Private"